File: app/src/main/java/com/stevenfrew/beatprompter/midi/ControlChangeMessage.java
package com.stevenfrew.beatprompter.midi;

class ControlChangeMessage extends OutgoingMessage {

    private static byte CONTROL_CHANGE_MESSAGE_BYTE=(byte)0xb0;
    static byte BANK_SELECT_MSB_BYTE=(byte)0;
    static byte BANK_SELECT_LSB_BYTE=(byte)32;

    ControlChangeMessage(byte controller, byte value, byte channel)
    {
        super(mergeMessageByteWithChannel(CONTROL_CHANGE_MESSAGE_BYTE,channel),controller,value);
    }
}

File: app/src/main/kotlin/com/stevenfrew/beatprompter/comm/midi/message/ControlChangeMessage.kt
package com.stevenfrew.beatprompter.comm.midi.message

internal class ControlChangeMessage(
	controller: Byte,
	value: Byte,
	channel: Byte
) :
	MidiMessage(mergeMessageByteWithChannel(MIDI_CONTROL_CHANGE_BYTE, channel), controller, value)